samples/presentation/add_video/cs/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Presentation;
+using A = DocumentFormat.OpenXml.Drawing;
+using P14 = DocumentFormat.OpenXml.Office2010.PowerPoint;
+using ShapeTree = DocumentFormat.OpenXml.Presentation.ShapeTree;
+using ShapeProperties = DocumentFormat.OpenXml.Presentation.ShapeProperties;
+using NonVisualDrawingProperties = DocumentFormat.OpenXml.Presentation.NonVisualDrawingProperties;
+using NonVisualPictureProperties = DocumentFormat.OpenXml.Presentation.NonVisualPictureProperties;
+using NonVisualPictureDrawingProperties = DocumentFormat.OpenXml.Presentation.NonVisualPictureDrawingProperties;
+using Picture = DocumentFormat.OpenXml.Presentation.Picture;
+using BlipFill = DocumentFormat.OpenXml.Presentation.BlipFill;
+using DocumentFormat.OpenXml.Packaging;
+using ApplicationNonVisualDrawingProperties = DocumentFormat.OpenXml.Presentation.ApplicationNonVisualDrawingProperties;
+
+// <Snippet0>
+AddVideo(args[0], args[1], args[2]);
+
+static void AddVideo(string filePath, string videoFilePath, string coverPicPath)
+{
+
+    string imgEmbedId = "rId4", embedId = "rId3", mediaEmbedId = "rId2";
+    UInt32Value shapeId = 5;
+    // <Snippet1>
+    using (PresentationDocument presentationDocument = PresentationDocument.Open(filePath, true))
+    // </Snippet1>
+    {
+
+        if (presentationDocument.PresentationPart == null || presentationDocument.PresentationPart.Presentation.SlideIdList == null)
+        {
+            throw new NullReferenceException("Presenation Part is empty or there are no slides in it");
+        }
+        // <Snippet2>
+        //Get presentation part
+        PresentationPart presentationPart = presentationDocument.PresentationPart;
```

Comment:
Do the samples not use `var`? Not a blocker, but we should be consistent

Reply:
I try to avoid var in the docs for clarity. Maybe we should go through older files and make them consistent

Reply:
we should add an editorconfig that enforces it for all of them